operator/slotticker/slotticker.go

```diff
@@ -0,0 +1,96 @@
+package slotticker
+
+import (
+	"time"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+)
+
+//go:generate mockgen -package=mocks -destination=./mocks/slotticker.go -source=./slotticker.go
+
+type Provider func() SlotTicker
+
+type SlotTicker interface {
+	Next() <-chan time.Time
+	Slot() phase0.Slot
+}
+
+type ConfigProvider interface {
+	SlotDurationSec() time.Duration
+	GetGenesisTime() time.Time
+}
+
+type Config struct {
+	slotDuration time.Duration
+	genesisTime  time.Time
+}
+
+func (cfg Config) SlotDurationSec() time.Duration {
+	return cfg.slotDuration
+}
+
+func (cfg Config) GetGenesisTime() time.Time {
+	return cfg.genesisTime
+}
+
+type slotTicker struct {
+	timer        *time.Timer
+	slotDuration time.Duration
+	genesisTime  time.Time
+	slot         phase0.Slot
+}
+
+// `New` returns a goroutine-free `SlotTicker` implementation which is not thread-safe.
```

Comment:
I have never seen function names wrapped in backticks in Godoc comments, so I'm not sure how it would be rendered and whether it would pass linter that checks comments for exported functions/methods. 

I ran a search for "// \`" in all SSV dependencies and didn't find any places where it's used in the beginning of a Godoc comment.

If backticks have any advantages, I don't mind, but in this case I think we need to discuss them, agree on them and to enforce this rule in all PRs to be consistent

Reply:
mb, it shouldn't begin with a backtick